### ISSUE TITLE
UNIX socket stops working after enabling HTTPS socket 

### ISSUE THREAD

**Issue by rrva** [lxc/lxd#1059]
If I enable the HTTPS socket and restart lxd, the UNIX socket stops working

```
lxc config set core.https_address [::]:8443
sudo systemctl restart lxd
lxc list
error: Get http://unix.socket/1.0: dial unix /var/lib/lxd/unix.socket: connect: no such file or directory
```

lxd.log

```
t=2015-08-29T12:29:59+0200 lvl=info msg="LXD is starting" path=/var/lib/lxd
t=2015-08-29T12:29:59+0200 lvl=info msg="Default uid/gid map:"
t=2015-08-29T12:29:59+0200 lvl=info msg=" - u 0 100000 65536"
t=2015-08-29T12:29:59+0200 lvl=info msg=" - g 0 100000 65536"
t=2015-08-29T12:29:59+0200 lvl=info msg=Init driver=storage/btrfs
t=2015-08-29T12:29:59+0200 lvl=info msg="Looking for existing certificates:" cert=/var/lib/lxd/server.crt key=/var/lib/lxd/server.key
t=2015-08-29T12:29:59+0200 lvl=info msg="LXD is socket activated"
t=2015-08-29T12:29:59+0200 lvl=info msg="REST API daemon:"
t=2015-08-29T12:29:59+0200 lvl=info msg=" - binding socket" socket=/var/lib/lxd/unix.socket
t=2015-08-29T12:29:59+0200 lvl=info msg=" - binding socket" socket=[::]:8443
```

If I disable systemd socket activation and restart lxd, it works again

```
systemctl stop lxd-unix.socket
systemctl disable lxd-unix.socket
systemctl restart lxd
```

This was on Ubuntu wily

**Comment by stgraber** [comment]
Sounds like some kind of race with systemd's socket stuff.

Note that you don't have to restart lxd after setting, unsetting or changing core.https_address.

**Comment by stgraber** [comment]
The systemd socket activation bug in question has been fixed in git as of yesterday, 0.18 will contain that fix.
